jekyll/_cci2/url-orbs.adoc

```diff
@@ -0,0 +1,173 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4+
+---
+= URL orbs
+:page-description: How to manage URL orbs allow-lists
+:experimental:
+:icons: font
+:page-layout: classic-docs
+
+[#introduction]
+== Introduction
+
+The ability for an organization to use private orbs is no longer dependent on publishing orbs. URL-referenced orbs offer a way to create orbs as well as control access within the organization.
+
+The use of URL orbs is recommended for private orbs where there are additional trust and access controls within the VCS.
+
+
+INFO:
```

Comment:
There are some formatting issues here when this page builds. 

The intro here can just intro the allow-lists "feature" with a short intro to what a URL orb is. The majority of the explanatory content for URL orbs is in the orbs-intro page.

So this intro can be to explain what an allow-list is and what this guide is going to show you how to do